.circleci/config.yml

```diff
@@ -9,6 +9,9 @@ jobs:
     steps:
       - checkout
       - run: python3 -m venv venv
+      - run: venv/bin/pip install git+https://github.com/sourcebots/sb-vision
```

Comment:
Could we pin this version

Reply:
Why? We're not actually using any part of it here, and it's only another place to have to update the version when we change things.

Reply:
i'm of the opinion that any external dependencies should be pinned. So why are we installing it?

Reply:
So that the `import sb_vision` in the tests works. There aren't actually any tests here which go near `sb_vision`, however things don't work if it's missing.